mne/source_space.py

```diff
@@ -35,69 +35,14 @@
                     _ensure_int, check_version, _get_call_line, warn,
                     _check_fname, _check_path_like, _check_sphere,
                     _validate_type, _check_option, _is_numeric, _pl, _suggest,
-                    object_size, sizeof_fmt)
+                    object_size, sizeof_fmt, read_freesurfer_lut)
 from .parallel import parallel_func, check_n_jobs
 from .transforms import (invert_transform, apply_trans, _print_coord_trans,
                          combine_transforms, _get_trans,
                          _coord_frame_name, Transform, _str_to_frame,
                          _ensure_trans, read_ras_mni_t)
 
 
-def read_freesurfer_lut(fname=None):
```

Comment:
Try to avoid these big namespace moves in PRs (unless they're dedicated just to that), it makes the diff/API discussion harder to review and discuss